sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluate/_evaluate_aoai.py

```diff
@@ -129,7 +123,10 @@ def _begin_single_aoai_evaluation(
     # Format data for eval group creation
     grader_name_list  = []
     grader_list = []
-    
+    # It's expected that all graders supplied for a single eval run use the same credentials
+    # so grab a client from the first grader.
+    client = list(graders.values())[0].get_client()
```

Comment:
get_client is that our internal implementation or aoai grader have such a method ?

Reply:
internal implementation - all of our grader wrapper classes have this function to produce a client from the inputted model config